src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -623,6 +626,7 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
                 now: mz_ore::now::SYSTEM_TIME.clone(),
                 refresh_before_secs: 60,
                 password_prefix: args.frontegg_password_prefix.unwrap_or_default(),
+                admin_role: args.frontegg_admin_role.unwrap_or_default(),
```

Comment:
`unwrap_or_default` here feels a bit scary due to rust determining whether a role is an admin or not, and we'd need to assure ourselves that frontegg doesn't accidentally create a empty string role, which would then be given admin privs.

Suggestion: make this an `Option<String>` in the frontegg config so we know if it's None, then not an admin. That will allow a week or so for the cloud team to add this cli arg to envd, and then we can require it in the future after we know all cloud envds are getting this arg passed to them.